clang/lib/CodeGen/CoverageMappingGen.cpp

```diff
@@ -1709,6 +1730,13 @@ struct CounterCoverageMappingBuilder
             : addCounters(
                   addCounters(ParentCount, BackedgeCount, BodyBC.ContinueCount),
                   IncrementBC.ContinueCount);
+    auto [ExecCount, ExitCount] =
+        (llvm::EnableSingleByteCoverage
```

Comment:
I thought I could introduce a temporary wrapper at first (and I tried once).
I concluded I could write naked dirty statements here, since they would be swept cleanly later.

Do you think we should introduce a wrapper anyways? I didn't want to focus cleaning up existing code.

Reply:
if it's going to change further, then this is fine as-is